src/components/ens-profile/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Text as NativeText } from 'react-native';
+import AvatarCoverPhotoMaskSvg from '../../svg/AvatarCoverPhotoMaskSvg';
+import { BackgroundProvider, Box, Cover } from '@rainbow-me/design-system';
+import { ImgixImage } from '@rainbow-me/images';
+
+const size = 70;
+
+export default function Avatar({
```

Comment:
maybe let's change the name for this one? we have another Avatar in `src/components/ens-registration/Avatar/Avatar.tsx`